entente/landmarks/landmark_composite_recipe.py

```diff
@@ -97,15 +96,18 @@ def composite_landmarks(self):
         )
         for example in self.examples:
             example_mesh = lacecore.load_obj(example["mesh"], triangulate=True)
-            landmarks = [
-                {"name": name, "point": example[name]} for name in self.landmark_names
-            ]
+            landmarks = {k: example[k] for k in self.landmark_names}
             compositor.add_example(mesh=example_mesh, landmarks=landmarks)
-        return compositor.result
+        return {
+            k: np.round(v, decimals=self.decimals) for k, v in compositor.result.items()
+        }
```

Comment:
These were not being rounded previously, but other results were, so I figured we should be consistent and round all the results